File: sw01/MyHelloWorld/app/src/main/java/com/example/myhelloworld/MainActivity.java
package com.example.myhelloworld;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {
    private static final String TAG = "MainActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        if (savedInstanceState == null) {
            Log.i(TAG, "savedInstanceState == null");
        } else {
            Log.i(TAG, savedInstanceState.toString());
        }

    }
}

File: sw04/Persistenz/app/src/main/java/com/example/persistenz/MainActivity.kt
package com.example.persistenz

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import com.example.persistenz.filesystem.OverviewFragment


class MainActivity : AppCompatActivity(R.layout.activity_main) {
    private val TAG = "MainActivity"


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        supportFragmentManager
            .beginTransaction()
            .add(R.id.mainView_replaceFrame, OverviewFragment())
            .addToBackStack("Main")
            .commit()
    }

    override fun onResume() {
        super.onResume()
        Log.d(TAG, "onResume")
    }




}
